main.go

```diff
@@ -154,7 +156,16 @@ func main() {
 		os.Exit(1)
 	}
 
-	kongCtx.FatalIfErrorf(kongCtx.Run(ctx, client))
+	err = kongCtx.Run(ctx, client)
+	if err != nil {
+		if k8serrors.IsForbidden(err) {
+			kongCtx.Fatalf("Permission denied: are you part of the org?")
```

Comment:
one minor improvement suggestion:

```go
if err := kongCtx.Run(ctx, client); err != nil {
    if k8serrors.IsForbidden(err) {
        err = errors.New("got permission denied error: are you part of the organization?")
    }
    kongCtx.FatalIfErrorf(err)
}
```

One thing though which might be a bit problematic: we loose the original permission error. Maybe we can think about some `--debug` flag which would show the original error in addition?